### PR TITLE
Unit Tests for Climate Data Parsing Module

### PR DESCRIPTION
**Pull request by Yaser-123**
Description:
We need to improve the reliability and robustness of our climate data parsing module by adding comprehensive unit tests. This will ensure that the module correctly parses various climate data formats and handles edge cases gracefully.
Tasks:
1.Set Up Testing Framework
2.Identify Key Functions for Testing
3.Write Unit Tests
4.Run and Verify Tests

Issue #131 

https://github.com/dhairyagothi/50_days_50_web_project/assets/140265777/07e4104c-f11d-44fe-977a-2b1f244354f6

https://github.com/dhairyagothi/50_days_50_web_project/assets/140265777/79fe2fd6-2537-451d-bb9a-94b7414b0684

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -283,6 +283,11 @@
             <div class="project-name">Netflix Landing Page Clone</div>
             <a href="Netflix_Cloning/Index.html" class="demo-link">Demo</a>
         </div>
+         <div class="table">
+            <div class="day-number">Day 51</div>
+            <div class="project-name">ClimaCode</div>
```
Comment (dhairyagothi) [inline public/index.html:293]: i recently merged a PR you add it as 52

**Review by dhairyagothi — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 1 above)
Comment (dhairyagothi) [reply public/index.html:293]: image

**Comment by Yaser-123** [comment]
Done changed as 52

**Comment by dhairyagothi** [comment]
always add email id yrr
dont remove PR format while making PR

**Comment by Yaser-123** [comment]
Email :- 1ammar.yaser@gmail.com
